ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/application_traffic_recognition.schema.yml

```diff
@@ -0,0 +1,134 @@
+                type: str
+                description: Ip prefix to match against the source/destination IP in packets (ex- 1.2.3.0/24).
+      user_defined_ipv4_applications:
+        type: list
+        description: List of user defined ipv4 applications.
+        primary_key: name
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Name of the application.
+            src_prefix_set_name:
+              type: str
+              description: Specify a defined prefix set name.
+            dest_prefix_set_name:
+              type: str
+              description: Specify a defined prefix set name.
+            protocol:
+              type: str
+              description: Protocol to match the packets against.
+            src_port_set_name:
+              type: str
+              description: Specify a defined port range name. Used to match against the src port in packets.
+            dest_port_set_name:
+              type: str
+              description: Specify a defined port range name. Used to match against the dest port in packets.
+      application_profiles:
+        type: list
+        description: Profiles is a collection of applications and categories with/without services.
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Name of the application profile.
+            applications:
+              type: list
+              description: Specify list of application and service pair to be part of the application profile.
+              items:
+                type: dict
+                keys:
+                  name:
+                    type: str
+                    description: Name of an application.
+                  service:
+                    type: str
+                    description: One of the supported services of the application.
+                    valid_values: ["audio-video", "chat", "default", "file-transfer", "networking-protocols", "peer-to-peer", "software-update"]
+            transports:
+              type: list
+              description: List of transport protocols.
+              items:
+                type: str
+                description: Transport name.
+                valid_values: ["http", "https", "udp", "tcp", "ip", "ip6", "ssl", "rtp", "sctp", "quic"]
+            categories:
+              type: list
+              description: Categories under this application profile.
+              items:
+                type: dict
+                keys:
+                  name:
+                    type: str
+                    description: Name of a category.
+                  service:
+                    type: str
+                    description: Service name. Matches applications supporting this service.
+                    valid_values: ["audio-video", "chat", "default", "file-transfer", "networking-protocols", "peer-to-peer", "software-update"]
```

Comment:
turns out the list of services is different per application based on some DPI engine and so there is a mapping between appliucation name and what service is valid but it may be too hard to map.
So in the description lets indicate that not all applications support all services and the user need to check.